package-lock.json

```diff
@@ -29156,25 +29156,27 @@
       }
     },
     "node_modules/playwright": {
-      "version": "1.36.1",
-      "resolved": "https://registry.npmjs.org/playwright/-/playwright-1.36.1.tgz",
-      "integrity": "sha512-2ZqHpD0U0COKR8bqR3W5IkyIAAM0mT9FgGJB9xWCI1qAUkqLxJskA1ueeQOTH2Qfz3+oxdwwf2EzdOX+RkZmmQ==",
+      "version": "1.40.1",
```

Comment:
Yea you can see here that below under blazor that mutliple different versions of playwright are being resolved between packages. Should be fixed by pinning in all packages as described in the other comment